nx/lib/nx/random.ex

```diff
@@ -129,33 +129,6 @@ defmodule Nx.Random do
     |> Nx.as_type({:u, 32})
   end
 
-  defnp random_bits(key, opts \\ []) do
-    assert_key!(key)
-    opts = keyword!(opts, shape: {}, bit_width: 32)
-    shape = opts[:shape]
-    bit_width = opts[:bit_width]
-
-    case bit_width do
-      64 ->
-        bits =
-          threefry2x32(key, Nx.iota({Nx.size(shape) * 2}))
-          |> Nx.reshape({2, :auto})
-          |> Nx.as_type({:u, 64})
-
-        bits = bits[0] <<< 32 ||| bits[1]
-
-        Nx.reshape(bits, shape)
-
-      32 ->
-        threefry2x32(key, Nx.iota(shape, type: {:s, 64}))
-        |> Nx.as_type({:u, 32})
-
-      _ ->
-        threefry2x32(key, Nx.iota(shape, type: {:s, 64}))
-        |> Nx.as_type({:u, bit_width})
-    end
-  end
```

Comment:
No changes here, I just moved it down.

Reply:
The rest of the changes in the file is to require fewer `as_type`. No changes to the algo.